### PR TITLE
7 August Updates

### PR DESCRIPTION
**Pull request by gabriel-seow**
- PK
- GW
- JP
- MZ
- GM
- DO
- DE
- US
- OM
- FR
- MO

### CODE REVIEW

File: src/constants/travel.yaml [hunk 1]
```diff
@@ -1734,14 +1737,22 @@ countries:
...
   GW:
+    reopening:
+      '2020-08-01':
     sources:
       domestic:
         - 'Inter-region travel is prohibited, schools are closed, public transport only operable at 50% capacity and face coverings are mandatory until July 25, 2020](https://gw.usmission.gov/covid-19-information/).'
+        - 'Authorities previously lifted the nationwide curfew which [ran between 8 pm and 6 am (local time)](https://www.garda.com/crisis24/news-alerts/366606/guinea-bissau-authorities-extend-state-of-emergency-to-august-24-update-7).'
       inbound:
-        - 'Entry and exit to and from Guinea-Bissau [permitted only by presenting a certificate of negative COVID-19 test results](https://gw.usmission.gov/covid-19-information/).'
+        - 'As of August 1, 2020, international flights at Osvaldo Vieira International Airport (OXB) have resumed operations.](https://gw.usmission.gov/covid-19-information/)'
```
Comment (jannwong) [inline src/constants/travel.yaml:1747]: missing `[`

File: src/constants/travel.yaml [hunk 2]
```diff
@@ -1734,14 +1737,22 @@ countries:
...
+      outbound:
+        - 'As of August 1, 2020, international flights at Osvaldo Vieira International Airport (OXB) have resumed operations.](https://gw.usmission.gov/covid-19-information/)'
```
Comment (jannwong) [inline src/constants/travel.yaml:1749]: missing `[`

File: src/constants/travel.yaml [hunk 3]
```diff
@@ -2985,6 +2996,19 @@ countries:
...
       outbound: 'no'
+  MO:
+    sources:
+      domestic:
+        - '[Domestic travel is allowed](https://www.macaotourism.gov.mo/en/announcement).'
+        - 'Wearing a mask is mandatory when traveling on public transport, including buses, taxis, the Light Rapid Transit (LRT) trains, and ferries. You will [not be allowed to board if you do not wear a mask](https://www.macaotourism.gov.mo/en/announcement).'
+      inbound:
+        - 'Residents of Macau and travelers from Mainland China, Hong Kong, and Taiwan are allowed entry under certain conditions. [All other foreign travelers are denied entry](https://www.garda.com/crisis24/news-alerts/364586/mongolia-authorities-extend-covid-19-measures-through-august-31-update-12).'
+      outbound:
+        - '[Many airlines have [reduced or canceled flights to Macau](https://mn.usembassy.gov/covid-19-information/).'
```
Comment (jannwong) [inline src/constants/travel.yaml:3007]: double `[`

File: src/constants/travel.yaml [hunk 4]
```diff
@@ -3561,7 +3586,8 @@ countries:
...
-        - 'A nationwide nightly curfew was imposed on July 25, 2020, between 7 pm and 6 am (local time), which includes a ban on all movement and the closure of all public spaces and shops [until at least August 8, 2020](https://www.garda.com/crisis24/news-alerts/364066/oman-authorities-extend-eid-al-adha-holiday-from-july-30-august-6-following-spike-in-covid-19-cases-update-19).'
+        - 'A nationwide nightly curfew was imposed on July 25, 2020, between 9 pm and 5 am (local time), which includes a ban on all movement and the closure of all public spaces and shops [until at least August 8, 2020](https://www.garda.com/crisis24/news-alerts/364066/oman-authorities-extend-eid-al-adha-holiday-from-july-30-august-6-following-spike-in-covid-19-cases-update-19).'
+        - 'The domestic ban on inter-state movement will be lifted on August 8, 2020](https://www.garda.com/crisis24/news-alerts/364066/oman-authorities-extend-eid-al-adha-holiday-from-july-30-august-6-following-spike-in-covid-19-cases-update-19).'
```
Comment (jannwong) [inline src/constants/travel.yaml:3590]: missing `[`

File: src/constants/travel.yaml [hunk 5]
```diff
@@ -4886,6 +4913,7 @@ countries:
...
         - 'As of July 27, 2020, passengers arriving in Washington, DC from a high-risk state (within the prior 14 days) who were traveling for non-essential activities will be required to [self-quarantine for 14 days](https://www.garda.com/crisis24/news-alerts/364271/us-arriving-passengers-in-massachusetts-and-washington-dc-to-undergo-14-day-quarantine-as-of-july-27-update-79).'
         - 'As of August 1, 2020, passengers arriving in Massachusetts will have to [undergo a mandatory 14-day quarantine](https://www.garda.com/crisis24/news-alerts/364271/us-arriving-passengers-in-massachusetts-and-washington-dc-to-undergo-14-day-quarantine-as-of-july-27-update-79).'
+        - 'A 14-day quarantine mandate is in place for travelers [arriving from 35 designated states](https://www.garda.com/crisis24/news-alerts/364271/us-arriving-passengers-in-massachusetts-and-washington-dc-to-undergo-14-day-quarantine-as-of-july-27-update-79).'
```
Comment (jannwong) [inline src/constants/travel.yaml:4916]: i think this is specifically for massachusetts and DC. we should state it. currently the sentence means all states will have this quarantine mandate

Comment (gabriel-seow) [reply src/constants/travel.yaml:4916]: ah this is actuallyl for new york, my bad on missing that out, but edited!

**Pull request merged**